src/components/PersonItem/PersonItem.tsx

```diff
@@ -0,0 +1,43 @@
+import { useParams } from 'react-router-dom';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink';
+import classNames from 'classnames';
+import { useContext } from 'react';
+import { PeopleContext } from '../../peopleContext';
+import { getPersonByName } from '../../utils/getPeopleByName';
+
+type Props = {
+  person: Person;
+};
+
+export const PersonItem: React.FC<Props> = ({ person }) => {
+  const { slug: userId, sex, born, died, fatherName, motherName } = person;
```

Comment:
This line is destructuring many properties from the `person` object. While this is not wrong, it could lead to confusion if more properties are added in the future. A better approach would be to use the `person` object directly in the JSX. This would make it clear where each property is coming from.